src/xmipp/bindings/python/xmipp_base.py

```diff
@@ -98,7 +110,321 @@ def tryRun(self):
             import traceback
             traceback.print_exc(file=sys.stderr)
             return 1
-            
+
+    @classmethod
+    def getModel(cls, *modelPath, **kwargs):
```

Comment:
why do you accept cls when you don't use it?

Reply:
I guess that just to allow for both self.getModel() and XmippScript.getModel()

Reply:
That sounds like `@staticmethod`

Reply:
@staticmethod would remove the cls.

Reply:
Done!